blog/2025-02-21/index.md

```diff
@@ -0,0 +1,124 @@
+
+- Static permissions and their impact on safety
+- Open Age Rating Service rating and details
+- If an app uses outdated runtimes
+- Release notes for each release
+- If static permissions increase between releases
+
+Flathub.org and GNOME Software also display the app’s verified status.
+
+## Updates
+
+Once an app is accepted onto Flathub, it’s not just the wild west; there are still a number of safety protections built into the flow:
+
+- **Flathub maintains ownership over the manifest repo**, while app developers are invited as limited collaborators
+- **The manifest’s default branch is protected**, preventing direct pushes without a pull request
+- **The manifest’s commit history cannot be rewritten**, making it harder to sneak something in
+- **Flathub’s automated tests must pass** before a PR can be merged and an update can be pushed
+- **Static permission changes are held for human review** before an update is released to users
+- **Critical MetaInfo changes are held for human review**, e.g. if an app name, developer name, app summary, or license changes
+
+## Caveats
+
+There are a few caveats to some of the points above which I would be remiss not to mention. First, Flathub has granted a small handful of trusted partners (including Mozilla and OBS Studio) the ability to directly upload their builds from their own infrastructure. Even for the few cases of direct uploads, we require a public manifest and build pipeline to enable similar reproducibility and auditability as outlined above. We also require the apps to be verified, and still run automated tests such as our linter against them.
+
+Lastly, some apps (around 6%) use [extra-data](https://docs.flatpak.org/en/latest/conventions.html#exporting-through-extra-data) to instruct Flatpak to download and unpack an existing package (e.g. a Debian package) during installation. These are largely proprietary apps that cannot be built on Flathub’s infrastructure, or apps using complex toolchains that require network access during build. This is discouraged since it does not enable the same level of auditability nor multi-architecture support that building from source does. As a result, this is heavily scrutinized during human review and only accepted as a last resort.
+
+Even with the above, the vast majority of apps are built reproducibly from source on Flathub’s infrastructure—and the handful that aren’t still greatly benefit from the transparency and auditability built into all of the other layers.
+
+## Incident Response
+
+While we expect to catch the vast majority of safety issues with the above, we are also able to respond to anything that may have slipped through. For example, we have the ability to quickly drop an app from the remote in case we find it’s malicious; we can also revert, drop, or veto changes made to apps.
```

Comment:
```suggestion
While we expect to catch the vast majority of safety issues with the above, we are also able to respond to anything that may have slipped through. For example, we have the ability to remove an app from the Flathub remote in case we find that it’s malicious. We can also revert, recall or block broken or malicious app updates.
```